tempesta_fw/t/functional/helpers/chains.py

```diff
@@ -33,6 +33,27 @@ def make_502_expected():
     )
     return response
 
+def response_500():
+    date = deproxy.HttpMessage.date_time_string()
+    headers = [
+        'Date: %s' % date,
+        'Content-Length: 0',
+        'Connection: keep-alive'
+    ]
+    response = deproxy.Response.create(status=500, headers=headers)
+    return response
+
+def response_502():
```

Comment:
There already  is https://github.com/tempesta-tech/tempesta/blob/390410117b8dd155c722b1f0f434437d43bb18ae/tempesta_fw/t/functional/helpers/chains.py#L28 That is the difference?

Reply:
it's the same. fixed